nle/env/base.py

```diff
@@ -481,113 +281,118 @@ def step(self, action: int):
                   `end_status`, i.e. a status info -- death, task win, etc. --
                   for the terminal state).
         """
-        response, done, info = self.env.step(self._actions[action])
-        response, done, info = self._perform_known_steps(response, done, info)
+        observation, done = self.env.step(self._actions[action])
+        observation, done = self._perform_known_steps(observation, done)
 
         self._steps += 1
 
-        last_response = self.response
-        self.response = response
+        last_observation = self.last_observation
+        self.last_observation = observation
```

Comment:
I do like this rename

Reply:
It used to be the "response" we got from RPC. Now it's just the observation directly in memory. :)